volttron/platform/control.py

```diff
@@ -840,6 +840,131 @@ def list_agent_rpc_code(opts):
     print_rpc_methods(opts, peer_method_metadata, code=True)
 
 
+def list_remotes(opts):
+    """ Lists remote certs and credentials.
+    Can be filters using the '--status' option, specifying
+    pending, approved, or denied.
+    The output printed includes:
+        user id of a ZMQ credential, or the common name of a CSR
+        remote address of the credential or csr
+        status of the credential or cert (either APPROVED, DENIED, or PENDING)
+
+     """
+    conn = opts.connection
+    if not conn:
+        _stderr.write("VOLTTRON is not running. This command "
+                      "requires VOLTTRON platform to be running\n")
+        return
+
+    output_view = []
+    try:
+        pending_csrs = conn.server.vip.rpc.call(AUTH, "get_pending_csrs").get(timeout=4)
+        for csr in pending_csrs:
+            output_view.append({"entry": {"user_id": csr["identity"],
+                                          "address": csr["remote_ip_address"]},
+                                "status": csr["status"]
+                                })
+    except TimeoutError:
+        print("Certs timed out")
+    try:
+        approved_certs = conn.server.vip.rpc.call(AUTH, "get_authorization_approved").get(timeout=4)
+        for value in approved_certs:
+            output_view.append({"entry": value, "status": "APPROVED"})
+    except TimeoutError:
+        print("Approved credentials timed out")
+    try:
+        denied_certs = conn.server.vip.rpc.call(AUTH, "get_authorization_denied").get(timeout=4)
+        for value in denied_certs:
+            output_view.append({"entry": value, "status": "DENIED"})
+    except TimeoutError:
+        print("Denied credentials timed out")
+    try:
+        pending_certs = conn.server.vip.rpc.call(AUTH, "get_authorization_failures").get(timeout=4)
+        for value in pending_certs:
+            output_view.append({"entry": value, "status": "PENDING"})
+    except TimeoutError:
+        print("Pending credentials timed out")
+
+    if len(output_view) == 0:
```

Comment:
Nit. You can leverage Python's idea of "truthy" and "falsy" to be more Pythonic. See [Python docs for more info](https://docs.python.org/3/library/stdtypes.html#truth-value-testing). In this case, an empty list is "falsy" and thus the if statement can be rewritten as:

```python

if not output_view:
   print("No remote certificates or credentials")
   return
```

Reply:
Good point! Fixed.